reposcan/pkgtree.py

```diff
@@ -182,14 +187,22 @@ def add_package_entry(self, pkg_id, name_id, evr_id, arch_id):
                                    }
         self.datadict[name_id].append(self.packagedata[pkg_id]['data'])
 
-    def associate_repos(self):
+    def _associate_repos(self):
         """Add repodata to the package entries"""
         with self._named_cursor() as cursor:
             cursor.execute("select pkg_id, repo_id from pkg_repo")
             for pkg_id, repo_id in cursor:
-                self.packagedata[pkg_id]['data']['repositories'].append(self.repodata[repo_id]['data'])
+                addrepo = {'lable': self.repodata[repo_id]['data']['label'],
```

Comment:
there's a typo, `lable` instead of `label`

Reply:
I see what I did... it hasn't been there forever.  I introduced it in one of my commits.  No longer could I just copy the "data" value (a dict of values for pkgtree output) to the output because for some repos I need to add module_name and module_stream to this output dict.  Now it builds a new output dict for each repo and copies the label, name, arch, releasever and revision values to it (and adds module_name and module_stream where appropriate)